sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -542,6 +558,10 @@ public Void visitReference(Reference symbol, SymbolToNotNullContext context) {
 
                 @Override
                 public Void visitFunction(Function symbol, SymbolToNotNullContext context) {
+                    if (isIgnoreThreeValuedLogicFunction(symbol)) {
+                        return null;
```

Comment:
Does this also work in case there is some distance between the NOT and the ignore3vl? Like `NOT (ignore3vl(...) AND ignore3vl(...))`